data/actions/lib/actions.lua

```diff
@@ -1,5 +1,5 @@
 function destroyItem(player, target, toPosition)
-	if target == nil or not target:isItem() then
+	if not target or not target:isItem() then
```

Comment:
I think you should change this to ```type(target) ~= "userdata"``` since there is a possibility of _target_ being a table.